hosting.html

```diff
@@ -0,0 +1,188 @@
+					will be shipped directly to your chosen provider. They will receive your Obelisks,
+					unpack them, connect them to power &amp; internet, then send you an email with information
+					on how to login remotely to your Obelisks so you can configure them and start mining.</p>
+
+					<h4>AVAILABLE PACKAGES</h4>
+					<p>The following hosting packages have been negotiated for you by Obelisk.  Soon, you will
+						be able to login to the Obelisk site and choose a provider.</p>
+						<table class="colo-table" width=100%>
+							<thead>
+								<tr class="colo-table-header">
+									<th>PROVIDER NAME</th>
+									<th>LOCATION</th>
+									<th>MONTHLY PRICE</th>
+									<th>PACKAGE DETAILS</th>
+								</tr>
+							</thead>
+							<tbody>
+								<tr>
+									<td>Syptec Krypto</td>
+									<td>Salt Lake City, Utah</td>
+									<td>$60/Obelisk, or<br>$450/8 Obelisks</td>
+									<td>
+										<ul>
+											<li>All power costs included</li>
+											<li>Remote hands included for reboots</li>
+											<li>Emergency Backup Power (3 days)</li>
+											<li>Secure data center specifically for mining</li>
+											<li>Additional services<ul>
+												<li>Private locked rack: $35/month</li>
+												<li>Miner cleaning/dusting: $20/request</li>
+											</ul>
+										</li>
+									</ul>
+								</td>
+							</tr>
+							<tr>
+								<td>Minery.tech</td>
+								<td>Smithville, Missouri</td>
+								<td>$50</td>
+								<td>
+								<ul>
+									<li>All power costs included</li>
+									<li>Remote hands included for reboots</li>
+									<li>No Emergency Backup Power</li>
+									<li>Smaller, but secure data center specifically for mining</li>
+									<li>Additional services<ul>
+										<li>Private locked rack: $35/month</li>
+										<li>Miner cleaning/dusting: $20/request</li>
+									</ul>
+								</li>
+							</ul>
+						</td>
+					</tr>
+				</tbody>		
+			</table>
+			<p>If you	know of a good hosting provider and you would like us to investigate adding an
+					approved package for the community (especially outside of the United States), please email
+					us at <a class="colo-table-contact-link" href="mailto:hello@obelisk.tech">hello@obelisk.tech</a>.</p>
+				<h4>BUSINESS RELATIONSHIP</h4>
+				<p>It is important to note that, although we have negotiated hosting packages with
+					the listed providers, your actual hosting agreement will be between you and the provider
+					directly, not through Obelisk.  Our role is simply to facilitate the connection between
+					you and them.  We have no other business or financial relationships with any of the
+					listed providers.</p>
+				</div>
+				<div class="separators col-md-12">
+					<div class="red-separator"></div>
+					<div class="separator"></div>
+				</div>
+			</div>
+		</div>
+		<div class="footer">
+			<ul class="terms">
```

Comment:
Indentation is off here

Reply:
I'll just beautify the whole file for consistency.